src/arduino_transports.c

```diff
@@ -5,11 +5,8 @@
 #include "teensy_transports.c.in"
 #elif defined(ARDUINO_ARCH_OPENCR)
 #include "opencr_transports.c.in"
-#else
-#error micro-ROS Library not supported for this platform
 #endif
```

Comment:
The transport is defined in the Arduino project and we don't need these files here. Should we move the OpenCR and Teensy transport files?